test/unit/eigensolver/CMakeLists.txt

```diff
@@ -12,19 +12,31 @@ add_subdirectory(mc)
 
 DLAF_addTest(test_band_to_tridiag
   SOURCES test_band_to_tridiag.cpp
-  LIBRARIES dlaf.eigensolver dlaf.core
+  LIBRARIES dlaf.core dlaf.eigensolver dlaf.factorization dlaf.solver
```

Comment:
This is the problem I was talking about few weeks ago during one of our meeting, about CMake dependencies and the fact that OBJECT libraries are not propagated as dependencies.

I don't know if I addressed the problem as you actually suggested, but in this way I had to manually propagate the new dependencies to all existing targets that were already using `dlaf.eigensolver`. Indeed, by adding the new implementation to this library, now it also has new dependencies (i.e. `dlaf.factorization`, `dlaf.solver`).